iocore/eventsystem/I_Thread.h

```diff
@@ -177,7 +177,7 @@ class Thread
 protected:
   Thread();
 
-  static ink_hrtime cur_time;
+  static std::atomic<ink_hrtime> cur_time;
```

Comment:
`Thread::cur_time` is shared across threads.

https://github.com/apache/trafficserver/blob/a5f647a2023d481600141fb124f93642c34e01c6/iocore/eventsystem/I_Thread.h#L159-L160